src/parsing/convertXDMReturnValue.ts

```diff
@@ -63,7 +63,7 @@ export default function convertXDMReturnValue<
 			if (!ebv.ready) {
 				throw new Error(`The expression ${expression} can not be resolved synchronously.`);
 			}
-			return ebv.value as IReturnTypes<TNode>[ReturnType.BOOLEAN];
+			return ebv.value as any;
```

Comment:
I think we can use `IReturnTypes<TNode>[TReturnType]` instead of the `any`s in this file.